terraform/scripts/acp-user

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------------------
+# Name: User Access Group Policies
+#
+# Description: Set the policies in an Access Group to allow user access to resources in
+# a Resource Group. Allow full access for using Kubernetes and OpenShift, but limited access
+# to existing services, and block creation of new services. Ideal for a system administrator
+# to grant access to a team of users.
+#
+# --------------------------------------------------------------------------------------------------------
+#
+# input validation
+if [ -z "$1" ]; then
+    echo "Usage: acp-user <ACCESS_GROUP> <RESOURCE_GROUP> <REGION>"
+    echo "Create the user Access Policies for an Access Group associated with a Resource Group"
+    exit
+fi
+
+ACCESS_GROUP=$1
+RESOURCE_GROUP=$2
+REGION=$3
+
+# input validation
+if [ -z "${ACCESS_GROUP}" ]; then
+    echo "Please provide the ACCESS_GROUP name"
+    exit
+fi
+
+# input validation
+if [ -z "${RESOURCE_GROUP}" ]; then
+    echo "Please provide the RESOURCE_GROUP name"
+    exit
+fi
+
+# input validation
+if [ -z "${REGION}" ]; then
+    echo "Please provide the REGION"
+    exit
+fi
+
+
+# Container registry does not currently support us-east region
+if [ "${REGION}" == "us-east" ]; then
+  echo "Changing region from us-east to us-south."
+  REGION="us-south"
+fi
+
+
+# Define the Polices for the Access Group, this are limited to usage patterns not administration patterns
+
+# Container Registry - 21
+# Grant access to use (push, pull, etc.) the environment's namespace in the image registry
+#ibmcloud iam access-group-policy-create ${ACCESS_GROUP} --roles Viewer,Operator,Reader --service-name container-registry --region ${REGION}
+ibmcloud iam access-group-policy-create ${ACCESS_GROUP} --roles Reader,Writer --service-name container-registry --region ${REGION} --resource-type namespace --resource ${RESOURCE_GROUP}
+
+# Kubernetes - 42
+# Grant access to use all clusters in the resource group, but not to create or delete clusters
+#ibmcloud iam access-group-policy-create ${ACCESS_GROUP} --roles Viewer,Operator,Editor,Reader,Writer,Manager  --service-name containers-kubernetes
+ibmcloud iam access-group-policy-create ${ACCESS_GROUP} --roles Viewer,Operator,Reader,Writer,Manager --resource-group-name ${RESOURCE_GROUP} --service-name containers-kubernetes
+
+# Kubernetes - 73 with IKS instance
+#ibmcloud iam access-group-policy-create ${ACCESS_GROUP} --roles Viewer,Operator,Editor,Reader,Writer,Manager --service-name containers-kubernetes --service-instance ${PREFIX}-iks-cluster
```

Comment:
We should delete all these commented out commands. The original ones will still be there in the Git history